cfgov/v1/models/learn_page.py

```diff
@@ -200,7 +200,7 @@ class EventPage(AbstractFilterPage):
                   "It can be obtained by clicking on Share > "
                   "Embed on Youtube.",
         validators=[
-            RegexValidator(regex='^https?:\/\/www\.youtube\.com\/embed\/.*$')
+            RegexValidator(regex=r'^https?:\/\/www\.youtube\.com\/embed\/.*$')
```

Comment:
Is this gonna require a migration?

Reply:
No, a change to the validator shouldn't.

Reply:
Confirmed with `tox -e missing-migrations`.